Mobile Buy SDK/Mobile Buy SDK/Models/BUYManagedObject.m

```diff
@@ -91,6 +92,83 @@ - (NSDictionary *)jsonDictionaryForCheckout
 	return self.JSONDictionary;
 }
 
+#pragma mark - Dynamic Property Accessor Resolution
+
+NS_INLINE NSString *KeyForSelector(SEL selector, BOOL *isSetter)
+{
+	NSString *name = NSStringFromSelector(selector);
+	BOOL setter = [name hasPrefix:@"set"];
+	if (isSetter) {
+		*isSetter = setter;
+	}
+	return setter ? [[name substringWithRange:NSMakeRange(3, name.length - 4)] lowercaseString] : name;
+}
+
+- (BOOL)respondsToSelector:(SEL)selector
+{
+	return [super respondsToSelector:selector] || [self isPropertyAccessor:selector];
+}
+
+static NSMutableDictionary *signatureCache;
```

Comment:
Is the signature cache actually necessary? The runtime does do some of its own caching but I'm not sure exactly when that happens and under what circumstances